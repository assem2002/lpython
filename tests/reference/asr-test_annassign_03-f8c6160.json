{
    "basename": "asr-test_annassign_03-f8c6160",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_annassign_03.py",
    "infile_hash": "aa787a87881a62595216dd4a7710bcf598550c100dda789794eb3c07",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_annassign_03-f8c6160.stderr",
    "stderr_hash": "90d7421450d9b2250a207a64611593df83c2cb1f42abcd68f2f7a28c",
    "returncode": 2
}