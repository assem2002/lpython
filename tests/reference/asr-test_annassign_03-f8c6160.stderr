semantic error: Intent annotation 'InOut' cannot be used here
 --> tests/errors/test_annassign_03.py:6:13
  |
6 |     lexer : InOut[LasrLexer] = LasrLexer(5)
  |             ^^^^^^^^^^^^^^^^ 


Note: Please report unclear or confusing messages as bugs at
https://github.com/lcompilers/lpython/issues.
